<?xml version="1.0"?>

<launch>

  <!-- realsense -->
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="depth_width"   value="640"/>
    <arg name="depth_height"  value="480"/>
    <arg name="enable_pointcloud"  value="true"/>
    <arg name="enable_sync"  value="true"/>
    <arg name="align_depth"  value="true"/>
  </include>

  <!-- point cloud realsense -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d /home/maggie/ros/rviz_config/realsense.rviz" />

  <!-- logitech -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video4" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui"/>

  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node> -->


</launch>